lib/mongo/monitoring.rb

```diff
@@ -25,6 +25,11 @@ class Monitoring
     # @since 2.1.0
     COMMAND = 'Command'.freeze
 
+    # The connection pool topic.
+    #
+    # @since 2.7.0
+    CONNECTION_POOL = 'ConnectionPool'.freeze
```

Comment:
This needs to be added to the tutorial under monitoring section with examples for how to subscribe to cmap events.

Reply:
Done